packages/EasyAsync/src/Bridge/Symfony/Resources/config/services.php

```diff
@@ -30,7 +30,7 @@
 
     $services->set(JobLogFactoryInterface::class, JobLogFactory::class);
 
-    $services->set(UuidGeneratorInterface::class, RamseyUuidGenerator::class);
+    $services->set(UuidV4GeneratorInterface::class, RamseyUuidV4Generator::class);
```

Comment:
Here, same, we should let the application decide which uuid v4 generator to use instead of making the decision for them, that's the point of using `easy-random`.

Reply:
Done.